src/Aspire.Hosting.Azure/Aspire.Hosting.Azure.csproj

```diff
@@ -42,6 +42,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Aspire.Hosting.Azure.Tests" />
     <InternalsVisibleTo Include="Aspire.Hosting.Azure.ContainerRegistry" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Azure.AppService" />
```

Comment:
@captainsafia @davidfowl - this is unfortunate. The reason we have it now is becuse we are making ARM calls using `ITokenCredentialProvider` and `azureEnvironment.ProvisioningContextTask` in `CheckWebSiteExistsAsync`.

Is there a better way to make these calls without using internals?